frontend/src/components/auth/PrivateRoute.tsx

```diff
@@ -1,19 +1,26 @@
 import React, { useContext } from "react";
 import { Route, Redirect } from "react-router-dom";
 
+import { Role } from "../../types/AuthTypes";
 import AuthContext from "../../contexts/AuthContext";
-import { LOGIN_PAGE, VERIFY_EMAIL_PAGE } from "../../constants/Routes";
+import {
+  HOME_PAGE,
+  LOGIN_PAGE,
+  VERIFY_EMAIL_PAGE,
+} from "../../constants/Routes";
 
 type PrivateRouteProps = {
   component: React.FC;
   path: string;
   exact: boolean;
+  roles?: ReadonlyArray<Role>;
 };
 
 const PrivateRoute: React.FC<PrivateRouteProps> = ({
   component,
   exact,
   path,
+  roles = ["Admin"],
```

Comment:
So the default "role" property of a page is such that it will only be visible to an admin? I guess this make sense? It seems like we should have the default role require to be none, and then explicit define restrictions on pages, but idk.

Reply:
Will leave a comment about this in the code.